pkg/cmd/cli/cmd/logs.go

```diff
@@ -189,5 +218,31 @@ func (o OpenShiftLogsOptions) RunLog() error {
 		// Use our own options object.
 		o.KubeLogOptions.Options = o.Options
 	}
-	return o.KubeLogOptions.RunLogs()
+	if !o.JenkinsLogs {
+		return o.KubeLogOptions.RunLogs()
+	} else {
+		bc, ok := o.KubeLogOptions.Object.(*buildapi.BuildConfig)
+		var urlString string
+		if !ok {
+			build, ok := o.KubeLogOptions.Object.(*buildapi.Build)
+			if ok {
+				urlString, _ = build.Annotations[buildapi.BuildJenkinsBlueOceanLogURLAnnotation]
+			}
+		} else {
+			buildName := buildutil.BuildNameForConfigVersion(bc.ObjectMeta.Name, int(bc.Status.LastVersion))
+			build, _ := o.Client.Builds(o.Namespace).Get(buildName, metav1.GetOptions{})
+			if build != nil {
+				urlString, _ = build.Annotations[buildapi.BuildJenkinsBlueOceanLogURLAnnotation]
+			}
+		}
+		if len(urlString) > 0 {
+			o.KubeLogOptions.Out.Write([]byte("Because this is a Jenkins Pipeline Strategy build, try accessing this URL:"))
+			o.KubeLogOptions.Out.Write([]byte("\n"))
+			o.KubeLogOptions.Out.Write([]byte(urlString))
+			o.KubeLogOptions.Out.Write([]byte("\n"))
+		} else {
+			return errors.New("Could not find the log URL for this Jenkins PIpeline Strategy build")
```

Comment:
fixed upper case I

Given that they just ran `oc logs` against a build or bc, having to provide the name, and can easily determine what namespace they are in via `oc project`, I do think that would be redundant.

Going to pass on that unless you argue further such that I'm convinced or give up resisting.